### ISSUE TITLE
Align prettify button to the left

### ISSUE THREAD

**Issue by Garbee** [devtools-html/debugger.html#1183]
We could move the prettify source button to the left to align with Chrome's layout. This would provide at least a little more consistency between our layouts which is useful to developers. Minor detail, but pleasant when our users don't need to search for functionality.

![selection_002](https://cloud.githubusercontent.com/assets/868301/20249499/51b1152e-a9c8-11e6-8dec-1a1e1a97c20f.png)

**Comment by jasonLaster** [comment]
Yup. I like that it's below the gutter as well. 

For historical reasons. It was in the right I think because in the first design it floated above the editor. 

Helen what do you think?

**Comment by helenvholmes** [comment]
Sounds good to me!

**Comment by jasonLaster** [comment]
Here's a patch to get started:

We do some funky things with the command bar and [RTL](https://github.com/jasonLaster/debugger.html/blob/60bf5585453a44a398cad056a936e529c4327f1d/docs/local-development.md#internationalization) positioning. 

```diff
diff --git a/public/js/components/SourceFooter.css b/public/js/components/SourceFooter.css
index 43ce230..2308ce7 100644
--- a/public/js/components/SourceFooter.css
+++ b/public/js/components/SourceFooter.css
@@ -13,11 +13,11 @@
 }

 html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+  float: left;
 }

 html[dir="rtl"] .source-footer .command-bar {
-  float: left;
+  float: right;
 }

 .source-footer .command-bar * {
@@ -35,11 +35,11 @@ html[dir="rtl"] .source-footer .command-bar {
 }

 html:not([dir="rtl"]) .command-bar > span {
-  margin-right: 0.7em;
+  margin-left: 0.7em;
 }

 html[dir="rtl"] .command-bar > span {
-  margin-left: 0.7em;
+  margin-right: 0.7em;
 }

 .source-footer .prettyPrint.pretty {
```

**Comment by Garbee** [comment]
Why are we floating instead of just using Flexbox?

**Comment by jasonLaster** [comment]
Good call. Flexbox would be better!

**Comment by Garbee** [comment]
I'll take a look at this once I have the stylelint fixes in place then.

**Comment by jasonLaster** [comment]
Sounds good

**Comment by Garbee** [comment]
Fixed by merging #1217.
